### PR TITLE
Bring back ReviewDB

### PR DESCRIPTION
**Pull request by mantikafasi**

### CODE REVIEW

**Comment by FieryFlames** [comment]
does anyone want to place bets on how long it takes to be removed again

**Comment by thororen1234** [comment]
yes 100% ill place a bet

**Comment by mantikafasi** [comment]
Reviewdb cycle is unescapeable

**Comment by cheesesamwich** [comment]
the cycle is infinite

**Review by zt64 — APPROVED** [review]

**Comment by mantikafasi** [comment]
I think this should get merged

**Review by vencordplus-a — APPROVED** [review]
I agree

**Review by Vendicated — APPROVED** [review]
i am joe biden and i approve this pull request 🇺🇸 🦅

**Comment by Vendicated** [comment]
just kidding i still need to review code i will do after lunch

File: src/plugins/ReviewDB/index.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    description: "Review other users (Adds a new settings to profiles)",
+    authors: [Devs.mantikafasi, Devs.Ven],
+
+    settings,
+
+    patches: [
+        {
+            find: "showBorder:null",
+            replacement: {
+                match: /user:(.{1,3}),setNote:.{1,3},canDM.+?\}\)/,
```
Comment (Vendicated) [inline src/plugins/ReviewDB/index.tsx:60]:
```suggestion
                match: /user:(\i),setNote:\i,canDM.+?\}\)/,
```

**Pull request merged**